src/shared.ts

```diff
@@ -1,10 +1,14 @@
 const EVENTS = {
+  CLOSE_APP: 'close-app',
+  CREATE_CAPTURE_WINDOW: 'create-capture-window',
   GET_ALL_DISPLAYS: 'get-all-displays',
   LOAD_PAGE: 'load-page'
 }
 
 const PAGES = {
-  MAIN: 'home'
+  MAIN: '/',
+  LOADING: '/loading',
+  CAPTURE: '/capture'
```

Comment:
refactor this with the web part to the `Record<string, ReactComponent>`, and use lazyLoad on ReactComponent.